test/index.js

```diff
@@ -101,6 +101,15 @@ describe("depcheck", function () {
     });
   });
 
+  it("should report bad javascript", function testBadJS(done) {
+    var absolutePath = path.resolve("test/fake_modules/bad_js");
+
+    depcheck(absolutePath, {  }, function checked(unused) {
+      assert.notEqual(unused.invalidFiles, {});
```

Comment:
I thought about that, but was worried about tweaking an existing test :3 I'll add it back